shared/js/background/GPC.es6.js

```diff
@@ -0,0 +1,57 @@
+/**
+ *
+ * Sets GPC signal
+ *
+ */
+const settings = require('./settings.es6')
+const utils = require('./utils.es6')
+
+// Return Sec-GPC header if setting enabled
+function getHeader () {
+    const GPCEnabled = settings.getSetting('GPCEnabled')
+    if (GPCEnabled) {
+        return {
+            name: 'Sec-GPC',
+            value: '1'
+        }
+    }
+}
+
+// Add navigator.globalPrivacyControl property to all frames if setting enabled
+// The script injected in Firefox is different from the script injected in Chromium-
+// based browsers due to slight differences in how the browsers interpret script-src
+// CSP directives, and the existence of Xray vision in Firefox.
+function injectDOMSignal (tabId) {
+    const supportedBrowsers = ['chrome', 'moz', 'edg']
+    let browserName = utils.getBrowserName()
+
+    if (!supportedBrowsers.includes(browserName)) return
```

Comment:
What browsers are unsupported? Maybe we should use chromium script as a default and only make an exception for 'moz'?

Reply:
TBH I'm not sure anything is unsupported, I put this in here more to prevent it from blowing anything up if for some reason `utils.getBrowserName()` returns an empty string (see https://github.com/duckduckgo/duckduckgo-privacy-extension/blob/develop/shared/js/shared-utils/parse-user-agent-string.es6.js#L19).

I'm happy to make chromium the default though, that makes sense to me (since only the FF content script has the potential to break things due to FF-specific code).

Reply:
Yep, I think we should go ahead with chromium script being the default.